tests/main/sql_chat/test_sql_chat_agent.py

```diff
@@ -120,6 +120,7 @@ def _test_sql_chat_agent(
     context: dict,
     prompt: str,
     answer: str,
+    use_schema_tools: bool = False,
```

Comment:
`use_schema_tools` is not being used anywhere in the code. Maybe the PR is missing some updates/files.

Reply:
The test and example fail because `use_schema_tools` is not defined in `SQLChatAgentConfig`. Probably because the PR is missing some updates.

Reply:
Yeah I was having some doubts about the test. The retrieval mode make it hard to tell on which turn the answer will be provided.

Reply:
yes we can figure out the right setting for turns, but for now the failures are for a different reason, it has to do with the `use_schema_tools`